### PR TITLE
Add entities.PartitionTable.update()

### PR DESCRIPTION
**Pull request by abalakh**
Added ```entities.PartitionTable.update()```, as it's needed for SatelliteQE/robottelo#2502.
Part of #111.

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -2774,6 +2778,17 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/ptables', 'server_modes': ('sat')}
         super(PartitionTable, self).__init__(server_config, **kwargs)
 
+    def update(self, fields=None):
+        """Fetch a complete set of attributes for this entity."""
```
Comment (Ichimonji10) [inline nailgun/entities.py]: Is there a bugzilla bug stating what's wrong? If not, can we make one? And if so, can we link to it?

My goal is to make sure that every single work-around in NailGun is associated with a bug, even when that means filing trivial bugs or bugs against old versions of Satellite. I'm tracking progress in #105. Feel free to add this method to that list. ([Some](https://bugzilla.redhat.com/show_bug.cgi?id=1230865) [examples](https://bugzilla.redhat.com/show_bug.cgi?id=1234973) [of](https://bugzilla.redhat.com/show_bug.cgi?id=1237257) [bugs](https://bugzilla.redhat.com/show_bug.cgi?id=1235379))

Comment (abalakh) [inline nailgun/entities.py]: Oops, my bad. Double checked - everything works just fine without any workarounds. 
API has confusing behavior sometimes - some entities require extra dict, some - don't, and some are working ok both with and without it.
I'll remove ```update()``` and ```update_payload()```. @Ichimonji10 , should i remove the tests as well? I can't see any tests for entities-with-no-workarounds.

Comment (Ichimonji10) [inline nailgun/entities.py]: > API has confusing behavior sometimes - some entities require extra dict, some - don't, and some are working ok both with and without it.

Yeah. It's a real bugger.

Yes, please do remove the tests from `test_entities.py` as well.

**Comment by Ichimonji10** [comment]
ACK

**Comment by oshtaier** [comment]
ACK

**Comment by elyezer** [comment]
:+1:

**Pull request merged**